index.js

```diff
@@ -19,6 +25,18 @@ const manageRecipes = async () => {
     await Recipe.deleteMany();
 
     // Run your code here, after you have insured that the connection was made
+
+  await Recipe.create({title:"Beans Soup", level:"easy peasy", ingredients:["beans"],cuisine:true, dishType:"soup",duration:10, creator:"Sheela Campos"})
```

Comment:
Everything is correct and works here, however, it would be a better practice if you stored your recipe object in a variable, and then, do the `Recipe.create()` method and pass the variable name as the argument.